### PR TITLE
List groceries

### LINKED ISSUE #43: Get Ingredients
- [x] Write a function to list Ingredients from chosen recipies.

- [x] total Ingredients if they are doubled

- [x] escape Ingredients you usally have at home like salt, oil, etc.

### LINKED ISSUE #53: Create an Icon for Recipe Overview

### PR DESCRIPTION
**Pull request by boldCM**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -119,10 +119,10 @@ app.get("/api/week", async (req, res) => {
...
 // Serve any static files
 app.use(express.static(path.join(__dirname, "client/build")));
-app.use(
-  "/storybook",
-  express.static(path.join(__dirname, "client/storybook-static"))
-);
+// app.use(
+//   "/storybook",
+//   express.static(path.join(__dirname, "client/storybook-static"))
+// );
```
Comment (lauravikanis) [inline server.js:125]: There might be a problem with the run() part of your function and that might be why its not deploying. Did you check your Database names in your .env? right now yours looks like:

> `async function run() {
>   console.log("Connecting to database...");
>   await connect(process.env.DB_URI, process.env.DB_DBNAME);
>   console.log("Connected to database ");`

  while mine looks like this: 

> `async function run() {
  console.log("Connecting to database...");
  await connect(process.env.MONGO_DB_URI, process.env.MONGO_DB_NAME);
  console.log("Connected to database ");`

**Review by lauravikanis — APPROVED** [review]
tried to solve your deployment problem but I'm not sure 🙃

File: client/src/components/BottomNav/BottomNav.js [hunk 2]
```diff
@@ -39,8 +41,14 @@ const BottomNav = () => {
           <HomeButton to="/">
             <img src={HomeIcon} alt={"Zur Wochenübersicht"} />
           </HomeButton>
-          <IconButton iconSrc={ShareIcon} iconAlt={"ShareIcon"} />
-          <IconButton iconSrc={DeleteIcon} iconAlt={"DeleteIcon"} />
+          <HomeButton to="/Einkaufsliste">
+            <img src={GroceryIcon} alt={"Zur Einkaufsliste"} />
+          </HomeButton>
+          <HomeButton to="/Rezeptübersicht">
+            <img src={RecipeIcon} alt={"Zur Rezeptübersicht"} />
```
Comment (mariegross) [inline client/src/components/BottomNav/BottomNav.js]:
```suggestion
            <img src={GroceryIcon} alt="Zur Einkaufsliste" />
          </HomeButton>
          <HomeButton to="/Rezeptübersicht">
            <img src={RecipeIcon} alt="Zur Rezeptübersicht" />
```
I don't think you need curly brackets if you type in the alt text as a string.

File: client/src/components/BottomNav/BottomNav.js [hunk 3]
```diff
@@ -39,8 +41,14 @@ const BottomNav = () => {
...
+          </HomeButton>
+          {/* <IconButton iconSrc={ShareIcon} iconAlt={"ShareIcon"} />
+          <IconButton iconSrc={DeleteIcon} iconAlt={"DeleteIcon"} /> */}
```
Comment (mariegross) [inline client/src/components/BottomNav/BottomNav.js]: If you don't need this anymore, I'd delete it :)

File: client/src/components/GroceryList/GroceryList.js [hunk 4]
```diff
@@ -33,48 +23,47 @@ const GroceryList = () => {
...
-  // const doubleArray = filterRecipe.Recipe;
-  // console.log(doubleArray);
-
-  const Array = findIngredients.Recipe.Ingredients;
-  console.log(Array);
-
-  // Für jedes Objekt in filterRecipe will ich die Ingredients darausgelesen haben.
-  // also für jedes Object einmal Array (Z.50) machen und dann unten mappen
-  // let arrayLength = recipeObject.length;
+  var holder = {};
```
Comment (mariegross) [inline client/src/components/GroceryList/GroceryList.js]: Would stay with `const` or `let` to keep true to your coding style :)

**Review by mariegross — APPROVED** [review]
LGTM, only small suggestions :)

File: server.js (hunk 1 above)
Comment (boldCM) [reply server.js:125]: thanks, problem was elserwhere ;)

File: client/src/components/BottomNav/BottomNav.js (hunk 2 above)
Comment (boldCM) [reply client/src/components/BottomNav/BottomNav.js]: thanks :)

File: client/src/components/GroceryList/GroceryList.js (hunk 4 above)
Comment (boldCM) [reply client/src/components/GroceryList/GroceryList.js]: oh yes, you're right!:)

**Pull request merged**